### PR TITLE
Fix the invalid args in assess_cloud_display.py

### PR DESCRIPTION
**Pull request by burton-swan**
## Description of change
Fix the failure observed in cloud display test:

http://ci.jujucharms.com/job/clouds-display/

## QA steps
$ export JUJU_HOME=<path_to>/cloud-city
$ export JUJU_BIN=<path_to>/juju
$ ./assess_cloud_display.py $JUJU_HOME/example-clouds.yaml $JUJU_BIN

This change has been locally validated.

## Documentation changes
N/A

## Bug reference
N/A

### CODE REVIEW

File: acceptancetests/assess_cloud_display.py [hunk 1]
```diff
@@ -123,4 +124,4 @@ def main():
 
 
 if __name__ == '__main__':
-    main()
+    sys.exit(main())
```
Comment (axw) [inline acceptancetests/assess_cloud_display.py]: main doesn't return anything, so why do this?

Comment (burton-swan) [reply acceptancetests/assess_cloud_display.py]: Ah, I keep it in align with other scripts, and I did a bit search, found this:

http://www.artima.com/weblogs/viewpost.jsp?thread=4829

If in future we make main() to return something, sys.exit() can capture it.

Comment (axw) [reply acceptancetests/assess_cloud_display.py]: fair enough

**Comment by burton-swan** [comment]
$$merge$$

**Pull request merged**